File: LeetCode/1486_XOR_Operation_in_an_Array/solution.js
/**
 * @param {number} n
 * @param {number} start
 * @return {number}
 */
var xorOperation = function(n, start) {
    let arr=[];
    let ans;
    for(let i=0;i<n;i++){
        arr[i]=start + 2 * i;
        if(i==0) ans=arr[0];
        else ans=ans^arr[i]
    }
    return ans;
};

File: LeetCode/1486_XOR_Operation_in_an_Array/solution.cpp
class Solution {
public:
    int xorOperation(int n, int start) {
        int arr;
        int ans = start;
        for (int i = 1; i < n; i++) {
            arr = start + 2 * i;
            if (ans == 0)
                ans = arr;
            else
                ans = ans ^ arr;
        }
        return ans;
    }
};